### PR TITLE
Add font feature fixes

### PR DESCRIPTION
**Pull request by wswebcreation**
This PR tries to fix a few cases where fonts result in flaky tests, these are:

- when 3rd party fonts are not constantly fully loaded
- font rendering between versions, or sometimes on the same versions

It implements:
- using `enableLayoutTesting` for all tests to reduce flakiness
- a complete new image baseline
- reduced workers that are used from 199 to 149
- reduced tests which should bring the test time under 25 min when no retries are executed (which was 25-35 min)

## 3rd party fonts flakiness
This is inconsistent and will not be fixed with a default check of the document ready state, 3rd party fonts are loaded after that. The solution is to verify if they are also loaded. This can be done with the asynchronous `document.fonts.ready` API.

This will be enabled by default but can be disabled on the service level. 

## Font rendering issues
In most cases, the font is not the most important thing on the page to check but can result in rendering issues that cause false negatives. This can be solved by not focusing on the text and "hiding" it by adding a `color: 'transparent'` to "each" element on the page.

The reasons for choosing the less performant 

```js
const elements = document.querySelectorAll<HTMLElement>('body *:not(script):not(style)')
elements.forEach(el => {
  el.style.setProperty('color', 'transparent', 'important')
})
```

instead of adding a stylesheet are:

- the stylesheet can be overruled by:
  -  inline styles
  - css selectors in the app style sheets that take priority over the stylesheet we want to inject
- all use cases of text in elements that are not `p, h1, h2, h3, h4, h5, h6, span, a, li`

The biggest downsides are (but accepted)
- icons will also be hidden
- it might be a performance intensive job

This will be disabled by default but can be enabled on the service and method option level.

> [!NOTE]
> This flag will introduce the "layout" testing option, where all text is hidden and only the layout is shown

The output will look like this

### element
wdioLayoutNavBar-chrome-latest-1366x768

### fullpage
fullPageLayout-chrome-latest-1366x768

### CODE REVIEW

File: tests/configs/sauce.ios.sims.web.ts [hunk 1]
```diff
@@ -96,40 +96,77 @@ export function sauceIosSimWeb({ buildName }: { buildName: string }) {
...
-        'iPad Pro (11 inch) (3rd generation) Simulator',
-        'iPad Pro (12.9 inch) (3rd generation) Simulator',
-        'iPad Pro (12.9 inch) (4th generation) Simulator',
-        'iPad Pro (12.9 inch) (5th generation) Simulator',
-    ]
+
+    /**
+     * We disabled the bezel devices for now. 7 devices are already checked in the iOS16Devices
+     */
+    // const iOS16BezelDevices = [
```
Comment (christian-bromann) [inline tests/configs/sauce.ios.sims.web.ts:137]: Should we remove this comment block or keep it for a specific purpose?

**Review by christian-bromann — COMMENTED** [review]
One comment on the comment block and whether we should keep that or remove.

File: tests/configs/sauce.ios.sims.web.ts (hunk 1 above)
Comment (wswebcreation) [reply tests/configs/sauce.ios.sims.web.ts:137]: I have a nasty habit of keeping those there, but will remove it. Will push in in a few min. Thanks

**Review by christian-bromann — APPROVED** [review]
LGTM 👍 

The failing build is flakiness?

**Comment by wswebcreation** [comment]
> LGTM 👍
> 
> The failing build is flakiness?

Yes 😢 , and it's not the module

**Pull request merged**